e2e/cases/cli/watch-files/index.test.ts

```diff
@@ -55,9 +56,9 @@ rspackOnlyTest(
       },
     });
 
-    await awaitFileExists(dist);
+    await awaitFileExists(distIndexFile);
 
-    fs.rmSync(dist, { recursive: true });
+    fs.rmSync(distIndexFile, { recursive: true });
```

Comment:
recursive is useless when removing a file

Reply:
```suggestion
    fs.rmSync(distIndexFile);
```